README.md

```diff
@@ -171,6 +171,39 @@ In the docs we'll try to explain both general concepts of MEAN components and gi
 ## Live Example
 Browse the live MEAN.JS example on [http://meanjs.herokuapp.com](http://meanjs.herokuapp.com).
 
+## Deploying To Cloud Foundry
+
+Cloud Foundry is an open source platform-as-a-service (PaaS).  The MEANJS project
+can easily be deployed to any Cloud Foundry instance.  The easiest way to deploy the
+MEANJS project to Cloud Foundry is to use a public hosted instance.  The two most popular
+instances are [Pivotal Web Services](https://run.pivotal.io/) and 
+[IBM Bluemix](https://bluemix.net).  Both provide free trials and support pay-as-you-go models
+for hosting applications in the cloud.  After you have an account follow the below steps to 
+deploy MEANJS.
+
+* Install the [Cloud Foundry command line tools](http://docs.cloudfoundry.org/devguide/installcf/install-go-cli.html).
+* Now you need to log into Cloud Foundry from the Cloud Foundry command line.
+  *  If you are using Pivotal Web Services run `$ cf login -a api.run.pivotal.io`.
+  *  If you are using IBM Bluemix run `$ cf login -a api.ng.bluemix.net`.
+* Create a Mongo DB service, IBM Bluemix and Pivotal Web Services offer a free MongoLabs service.
+  *  `$ cf create-service mongolab sandbox mean-mongo`
+* Clone the GitHub repo for MEANJS if you have not already done so
+  * `$ git clone https://github.com/meanjs/mean.git && cd mean`
+* Run the Grunt Build task to build the optimized JavaScript and CSS files
+  * `$ grunt build`
```

Comment:
@ryanjbaxter In this pull request you say you have to run grunt build, but in the pull request to the documentation you don't mention anything about grunt build. Is it required?